packages/order-utils/src/asset_data_utils.ts

```diff
@@ -1,10 +1,13 @@
-import { AssetData, AssetProxyId, ERC20AssetData, ERC721AssetData } from '@0x/types';
-import { BigNumber } from '@0x/utils';
-import ethAbi = require('ethereumjs-abi');
-import ethUtil = require('ethereumjs-util');
+import { AssetData, AssetProxyId, ERC20AssetData, ERC721AssetData, MultiAssetData } from '@0x/types';
+import { AbiEncoder, BigNumber } from '@0x/utils';
+import { MethodAbi } from 'ethereum-types';
+import * as _ from 'lodash';
 
 import { constants } from './constants';
 
+const encodingRules: AbiEncoder.EncodingRules = { optimize: true };
+const decodingRules: AbiEncoder.DecodingRules = { structsAsObjects: true };
```

Comment:
Can we rename these configs to follow our boolean naming conventions: `shouldOptimize` and `shouldConvertStructsToObjects`